src/mame/drivers/hornet.cpp

```diff
@@ -856,7 +896,7 @@ WRITE16_MEMBER(hornet_state::gn680_latch_w)
 }
 
 // WORD at 30000e: IRQ 4 tests bits 6 and 7, IRQ5 tests bits 4 and 5
-// (vsync and hsync status for each of the two screens?)
+// (X and Y values for both players analog controls?)
```

Comment:
Specific bit tests in interrupt handlers don’t sound like something that would be used for reading analog controls.  Do you have anything you’re basing this supposition on?